### ISSUE TITLE
yarn install gives an error

### ISSUE THREAD

**Issue by vitr** [zodman/django-sockpuppet-expo#7]
```
yarn install                                                 ✔  at 08:55:31 pm
yarn install v1.22.10
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
info There appears to be trouble with your network connection. Retrying...
error An unexpected error occurred: "https://nexus.gcds.coke.com/repository/npm-group/turbolinks/-/turbolinks-5.2.0.tgz: getaddrinfo ENOTFOUND nexus.gcds.coke.com".
```

**Comment by zodman** [comment]
damn! the registrant need update sorry for that ...

**Comment by zodman** [comment]
fixed at c66ee06a5feac2ce06d2d9e420c2c77b9a2c9b72
